### ISSUE TITLE
Navigation buttons click area extends across screen

### ISSUE THREAD

**Issue by hlb8122** [jhildenbiddle/docsify-themeable#15]
The navigation items are clickable all the way across the screen.

```
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsify-themeable@0/dist/css/theme-simple.css">
</head>
<body>
  <nav>
    <a href="http://www.google.com">Google clickable</a>
  </nav>
  <div id="app"></div>
  <script>
    window.$docsify = {
      homepage: 'introduction.md'
  }
  </script>
  <script src="//unpkg.com/docsify/lib/docsify.min.js"></script>
  <script src="https://cdn.jsdelivr.net/npm/docsify-themeable@0"></script>
</body>
</html>
```

**Comment by jhildenbiddle** [comment]
Hi @hlb8122.

This issue has been addressed and will be available 0.6.3 release.

It's worth noting that this issue only affects links manually added to a top-level `nav` element that are not contained within a `<li>` element. If you allow docsify to generate a navbar from markdown ([see documentation](https://docsify.js.org/#/custom-navbar)), this is not an issue. Regardless, I've updated the CSS so that the navbar link in your sample HTML will no longer have styles applied.

Thanks for taking the time to file the issue. The new release will be out before EOD.
